hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlan.java

```diff
@@ -141,12 +141,12 @@ public void initialize(NodeEngine nodeEngine, long jobId, long executionId, Snap
 
             int processorIdx = 0;
             for (Processor p : processors) {
-                ILogger logger = nodeEngine.getLogger(p.getClass().getName() + '.' + srcVertex.name()
-                                + '#' + (srcVertex.getProcIdxOffset() + processorIdx));
+                String loggerName = createLoggerName(p.getClass().getName(), srcVertex.name(),
```

Comment:
minor, but it might be good to have a method to calculate "global proc idx", I see this addition in three places now (twice inside execution plan) and one here.